docs/architecture.rst

```diff
@@ -66,53 +66,105 @@ pg_auto_failover Glossary
 
 pg_auto_failover handles a single PostgreSQL service with the following concepts:
 
-  - the pg_auto_failover MONITOR is a service that keeps track of one or several
-    *formations* containing *groups* of two *nodes* each.
+Monitor
+^^^^^^^
 
-    The monitor is implemented as a PostgreSQL extension, so when you run
-    the command ``pg_autoctl create monitor`` a PostgreSQL instance is
-    initialized, configured with the extension, and started. The monitor
-    service is embedded into a PostgreSQL service.
+The pg_auto_failover monitor is a service that keeps track of one or several
+*formations* containing *groups* of two *nodes* each.
 
-  - a FORMATION is a logical set of PostgreSQL services.
+The monitor is implemented as a PostgreSQL extension, so when you run the
+command ``pg_autoctl create monitor`` a PostgreSQL instance is initialized,
+configured with the extension, and started. The monitor service embeds a
+PostgreSQL instance.
 
-  - a GROUP of two PostgreSQL NODES work together to provide a single
-    PostgreSQL service in a Highly Available fashion. A GROUP consists of a
-    PostgreSQL primary server and a secondary server setup with Hot Standby
-    synchronous replication. Note that pg_auto_failover can orchestrate the whole
-    setting-up of the replication for you.
+Formation
+^^^^^^^^^
 
-  - the pg_auto_failover KEEPER is an agent that must be running on the same server
-    where your PostgreSQL nodes are running. The KEEPER controls the local
-    PostgreSQL instance (using both the ``pg_ctl`` command line tool and SQL
-    queries), and communicates with the MONITOR:
+A formation is a logical set of PostgreSQL services.
 
-      - it sends updated data about the local node, such as the WAL delta in
-        between servers, measured via PostgreSQL statistics views,
+.. note::
+   The notion of a formation in pg_auto_failover is useful when setting-up
```

Comment:
This note seems more related to groups than to formations. I would move it down one subsection and change it like this.

```suggestion
   The notion of a group in pg_auto_failover is useful when setting-up
```

Reply:
Well it's confusing. If it was not for Citus, we would not have the notion of a formation, I believe. Only groups. But we might have named what is currently a group a formation, that said.

Reply:
Yeah I guess we would only need one notion if not for Citus, but with non-Citus you can only have formations right? You cannot have a non-zero group at all with regulag PG afaik.

Reply:
That's a good point indeed. Moved to the group definition now.